xml/security_ldap.xml

```diff
@@ -316,10 +316,11 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
     </para>
  </sect1>
 
- <sect1 xml:id="sec-security-ldap-server" os="sles;osuse">
-  <title>Manually Configuring a &ds389;</title>
+ <sect1 xml:id="sec-security-ldap-server">
+  <title>Setting up a &ds389; Test Instance</title>
```

Comment:
Very minor, but could "development instance" be better than "test" here?

Reply:
how about 'testing and development'